Fundamentals/Data/Creating-Media/index-v9.md

```diff
@@ -0,0 +1,167 @@
+---
+meta.Title: "Creating media in Umbraco"
+meta.Description: "Media in Umbraco is handled in much the same way as content. From the backoffice you can upload and create media items, such as images and files."
+versionFrom: 8.14.0
+verified-against: 9.0.0
+---
+
+# Creating media
+
+Media in Umbraco is handled the same way as content. Instead of defining Document Types you define Media Types that acts as a base for media items. The following default Media Types are available:
+
+- Article - TBD
```

Comment:
As far I can see, this Media Type allows uploads of `.pfd`, `.docs` and `doc` files.
![image](https://user-images.githubusercontent.com/24529086/120158995-0336bf80-c1f5-11eb-9f69-814590f07998.png)

So, perhaps this line could be something like: "Used for uploading and storing documents"? 🤔